google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -217,25 +348,28 @@ def _get_blocks(blocks: List[documentai.Document.Page.Block], text: str) -> List
             from the document.
     Returns:
         List[Block]:
-             A list of Blocks.
+             A list of wrapped Blocks.
     """
     result = []
 
     for block in blocks:
         result.append(
             Block(
-                documentai_block=block,
+                documentai_object=block,
                 text=_text_from_layout(layout=block.layout, text=text),
+                _page=_page,
             )
         )
 
     return result
 
 
 def _get_paragraphs(
-    paragraphs: List[documentai.Document.Page.Paragraph], text: str
+    paragraphs: List[documentai.Document.Page.Paragraph],
+    text: str,
+    _page: Page,
```

Comment:
As an argument of a private helper function, let's not prefix the argument name with underscore.  Plain `page` for the argument name is fine.

(For the fields/attributes, we prefix with underscore to signal to the user that that field is private and since this is Python the user is free to access it, but we make no guarantees that API is table.)